### PR TITLE
Add service-level logging for remaining service classes

### PR DESCRIPTION
**Pull request by blurrymoi**

### CODE REVIEW

File: logging-service/src/main/java/cz/muni/pa036/logging/service/EventServiceImpl.java [hunk 1]
```diff
@@ -110,24 +123,31 @@ public void delete(Event event) {
...
+                resultDAO.delete(result);
+            });
         }
 
         //next delete all invitations for this event due to foreign key
+        CRUD_LOGGER.logFindBy("event", event);
         List<Invitation> invitationList = invitationDAO.findByEvent(event);
         if (invitationList != null) {
-            for(Invitation invitation : invitationDAO.findByEvent(event)) {
+            for(Invitation invitation : invitationList) {
```
Comment (kamil-triscik) [inline logging-service/src/main/java/cz/muni/pa036/logging/service/EventServiceImpl.java]: why not **invitationList.forEach** like above?

File: logging-service/src/main/java/cz/muni/pa036/logging/service/ResultServiceImpl.java [hunk 2]
```diff
@@ -18,73 +21,103 @@
...
     }
 
     @Override
     public Result findBySportsmanAndEvent(Sportsman sportsman, Event event) {
+
+        Map<Object, Object> findBy = new HashMap<>();
+        findBy.put("sportsman", sportsman);
+        findBy.put("event", event);
+
+        CRUD_LOGGER.logFindBy(findBy);
```
Comment (kamil-triscik) [inline logging-service/src/main/java/cz/muni/pa036/logging/service/ResultServiceImpl.java]: next time consider to do it this way :p

**Map findBy = new HashMap<Object, Object>(2) {{
&nbsp;&nbsp;&nbsp;&nbsp;put("sportsman", sportsman);
&nbsp;&nbsp;&nbsp;&nbsp;put("event", event);
}};**

Comment (f-gdovin) [inline logging-service/src/main/java/cz/muni/pa036/logging/service/ResultServiceImpl.java]: jesus fuck this is ugly :D

Comment (kamil-triscik) [inline logging-service/src/main/java/cz/muni/pa036/logging/service/ResultServiceImpl.java]: it is f*cking beautiful :D

Comment (blurrymoi) [inline logging-service/src/main/java/cz/muni/pa036/logging/service/ResultServiceImpl.java]: nemate tam nejaky overkill hunatych zatvoriek?

**Pull request merged**